### DISCUSSION TITLE
Is it possible to change 2fa setting after losing access to my 2fa application?

### DISCUSSION THREAD

**Discussion by tunztunztunz** [community/community#67665] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question - Authentication

### Body

I feel stupid, but my iPhone completely stopped working and I've lost access to my 2FA Microsoft authenticator application. I thought the app automatically made backups on iCloud, but apparently you had to do them manually, so now I'm screwed and can't recover all of my 2FA connections as far as their help documentation is concerned. 

I see that it seems unlikely in this case for Github, but is it possible for me to change my 2FA settings without access to my 2FA account while I still have access to my Github account? 

My worry is that I will eventually lose access if one of the trusted devices I've been using loses access for any reason.

**Comment by ValentinoDelicious66** [comment]
You can access in GitHub without password an mail

**Comment by ValentinoDelicious66** [comment]
Vad är det du försöker göra?? Skriv svenska ..
